flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/ResourceManagerTest.java

```diff
@@ -166,6 +166,34 @@ public void testRequestTaskManagerInfo() throws Exception {
         assertEquals(0, taskManagerInfo.getNumberAvailableSlots());
     }
 
+    /**
```

Comment:
Tests should be part of the respective main commits. You want each commit to stand on its own feet (that's not always strictly possible).